modules/candidate_profile/templates/form_candidate_profile.tpl

```diff
@@ -11,7 +11,7 @@
 window.addEventListener('load', () => {
     let candidate = null;
     async function loadCandidate() {
-        let response = await fetch(loris.BaseURL + '/api/v0.0.2/candidates/{$candidate->getCandID()}');
+        let response = await fetch(loris.BaseURL + '/api/v0.0.3/candidates/{$candidate->getCandID()}');
```

Comment:
@driusan this should be broken on latest release.

Reply:
There's already #7150 to fix this.. can you just review that?